models/models/cohort.py

```diff
@@ -52,6 +52,28 @@ class CohortCriteriaInternal(SMBase):
     sg_type: list[str] | None = None
     sample_type: list[str] | None = None
 
+    def to_external(self) -> dict:
+        """
+        Convert to external model
+        """
+        return {
+            'projects': [str(p) for p in self.projects] if self.projects else [],
```

Comment:
Should this be translating from numeric project id to textual project name?

Reply:
🤦‍♀️ Yes, thank you. I just converted 1 to "1" 😂  Let me fix that!!

Reply:
Ok I've pushed a solution to this, but I'm not convinced that it is a very elegant approach so happy to take feedback if you can see a better way.